utils/k8s/k8s.go

```diff
@@ -660,3 +660,22 @@ func (k8s K8S) BlockUntilPodIsUpOrTimeout(pod *core_v1.Pod, timeout time.Duratio
 
 	return k8s.BlockUntilPodIsUpWithContext(ctx, pod)
 }
+
+// CreatePersistentVolumeClaim is use to create the PVC in the given name space
+func (k8s K8S) CreatePersistentVolumeClaim(persistentVolumeClaim *core_v1.PersistentVolumeClaim, ns string) (result *core_v1.PersistentVolumeClaim, err error) {
+	persistentVolumeClaimNameSpace := k8s.Clientset.CoreV1().PersistentVolumeClaims(ns)
+	result, err = persistentVolumeClaimNameSpace.Create(persistentVolumeClaim)
+	return
```

Comment:
Can we avoid naked returns?
See following for reference
https://github.com/openebs/CITF/pull/53/files#diff-0838b5a1a898271846f11064eaefaab7R46

Reply:
And same for other cases.

Reply:
@farhaanbukhsh can you change ns to namespace as we're following the full name convention in parameters of methods.